src/views/AdvancedSearchView/__tests__/index.test.js

```diff
@@ -58,45 +59,42 @@ describe('AdvancedSearchView', () => {
     push: event => mockPush(event),
   };
 
-  let getByTestId;
-  let getByText;
-
-  beforeEach(() => {
-    ({
-      getByTestId, getByText,
-    } = render(
+  test('search button fires correctly', () => {
+    render(
       <AdvancedSearchView
```

Comment:
why did you move the render out of the beforeEach? (seems like the tests still all render the same)

Reply:
I guess it just didn't seem necessary. it only saves a few lines of code and tests are easier to read and debug when all their code is within the test block